ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -57,3 +57,70 @@ if have_tensorflow_lite
     install_dir: nnstreamer_libdir
   )
 endif
+
+filter_sub_python_sources = [
+  'tensor_filter_python.c',
+  'tensor_filter_python_api.c',
+  'tensor_filter_python_core.cc'
+]
+
+nnstreamer_filter_python_sources = []
+foreach s : filter_sub_python_sources
+  nnstreamer_filter_python_sources += join_paths(meson.current_source_dir(), s)
+endforeach
+
+if have_python
+  nnstreamer_filter_python_deps = [python_dep, libdl_dep, glib_dep, gst_dep, nnstreamer_dep]
+
+  shared_library('nnstreamer_filter_python',
+    nnstreamer_filter_python_sources,
+    cpp_args: ['-Wno-sign-compare'],
+    dependencies: nnstreamer_filter_python_deps,
+    install: true,
+    install_dir: filter_subplugin_install_dir
+  )
+
+  static_library('nnstreamer_filter_python',
+    nnstreamer_filter_python_sources,
+    cpp_args: ['-Wno-sign-compare'],
+    dependencies: nnstreamer_filter_python_deps,
+    install: true,
+    install_dir: nnstreamer_libdir
+  )
+
+  shared_library('nnstreamer_python',
+    'tensor_filter_python_api.c',
```

Comment:
The other shared library includes ```tensor_filter_python_api.c```. Why do you need this?

Reply:
Omg. it's mistake